examples/morpheusvm/tests/workload/genesis.go

```diff
@@ -72,7 +74,17 @@ func newDefaultAuthFactories() []chain.AuthFactory {
 
 func NewTestNetworkConfig(minBlockGap time.Duration) (workload.DefaultTestNetworkConfiguration, error) {
 	keys := newDefaultAuthFactories()
-	genesis := newGenesis(keys, minBlockGap)
+	testsLocalAllocations, err := tests.TestsRegistry.GenerateCustomAllocations(func() (chain.AuthFactory, error) {
+		privateKey, err := ed25519.GeneratePrivateKey()
+		if err != nil {
+			return nil, err
+		}
+		return auth.NewED25519Factory(privateKey), nil
+	})
```

Comment:
The lambda function could potentially be moved in `auth` package.

Reply:
Makes sense